# Genome-on-Silicon
A Collection of Papers on Genome Accelerators, from 2018 to Now.

---

## HPCA 2018
Searching for Potential gRNA Off-Target Sites for CRISPR/Cas9 Using Automata Processing Across Different Platforms, UVA

## ASPOLS 2018
Darwin: A Genomics Co-processor Provides up to 15000X Acceleration on Long Read Assembly, Stanford University

## ISCA 2018
GenAx: A Genome Sequencing Accelerator, UofM

## MICRO 2018
NONE

---

## HPCA 2019
Darwin-WGA: A Co-processor Provides Increased Sensitivity in Whole Genome Alignments with High Speed, Stanford University

FPGA Accelerated INDEL Realignment in the Cloud, UCB

## ASPLOS 2019
NONE

## ISCA 2019
NONE

## MICRO 2019
GenCache: Leveraging In-Cache Operators for Efficient Sequence Alignment, University of Utah

MEDAL: Scalable DIMM-based Near Data Processing Accelerator for DNA Seeding Algorithm, UCSB

---

## HPCA 2020
NONE

## ASPLOS 2020
NONE

## ISCA 2020
Genesis: A Hardware Acceleration Framework for Genomic Data Analysis, SNU

## MICRO 2020
SeedEx: A Genome Sequencing Accelerator for Optimal Alignments in Subminimal Space, UofM

GenASM: A High-Performance, Low-Power Approximate String Matching Acceleration Framework for Genome Sequence Analysis, ETHZ

---

## HPCA 2021
EXMA: A Genomics Accelerator for Exact-Matching, Indiana University Bloomington

## ASPLOS 2021
NONE

## ISCA 2021
Accelerated Seeding for Genome Sequence Alignment with Enumerated Radix Trees, UofM

Sieve: Scalable In-situ DRAM-based Accelerator Designs for Massively Parallel k-mer Matching, UVA

## MICRO 2021
SquiggleFilter: An Accelerator for Portable Virus Detection, UofM

---

## HPCA 2022
NONE

## ASPLOS 2022
GenStore: a high-performance in-storage processing system for genome sequence analysis, ETHZ

ProSE: the architecture and design of a protein discovery engine, Duke

## ISCA 2022
SeGraM: a universal hardware accelerator for genomic sequence-to-graph and sequence-to-sequence mapping, ETHZ

BioHD: an efficient genome sequence search platform using HyperDimensional memorization, UCI

EDAM: Edit Distance tolerant Approximate Matching content addressable memory, Technion-Israel Institute of Technology

## MICRO 2022
BEACON: Scalable Near-Data-Processing Accelerators for Genome Analysis near Memory Pool with the CXL Support, UCSB

GenPIP: In-Memory Acceleration of Genome Analysis via Tight Integration of Basecalling and Read Mapping, ETHZ

---

## HPCA 2023
NvWa: Enhancing Sequence Alignment Accelerator Throughput via Hardware Scheduling, ICT,CAS

## ASPLOS 2023
NONE

## ISCA 2023
GenDP: A Framework of Dynamic Programming Acceleration for Genome Sequencing Analysis, UofM

## MICRO 2023
Swordfish: A Framework for Evaluating Deep Neural Network-based Basecalling using Computation-In-Memory with Non-Ideal Memristors, ETHZ

DASH-CAM: Dynamic Approximate SearcH Content Addressable Memory for genome classification, Bar-Ilan University

CASA: An Energy-Efficient and High-Speed CAM-based SMEM Seeding Accelerator for Genome Alignment, THU

GMX: Instruction Set Extensions for Fast, Scalable, and Efficient Genome Sequence Alignment, Barcelona Supercomputing Center

---

## HPCA 2024
TALCO: Tiling Genome Sequence Alignment Using Convergence of Traceback Pointers, UCSD

## ASPLOS 2024
Harp: Leveraging Quasi-Sequential Characteristics to Accelerate Sequence-to-Graph Mapping of Long Reads, THU

## ISCA 2024
QUETZAL: Vector Acceleration Framework For Modern Genome Sequence Analysis, Barcelona Supercomputing Center

BLESS: Bandwidth and Locality Enhanced SMEM Seeding Acceleration for DNA Sequencing, KAIST

MegIS: High-Performance, Energy-Efficient, and Low-Cost Metagenomic Analysis with In-Storage Processing, ETHZ

## MICRO 2024
TBD...
